### PR TITLE
SDL requirement- package version update

### PR DESCRIPTION
**Pull request by waliMSFT**
-- SDL requirement: Open Source Components must be kept up to date and cannot be 18 months old
work item: https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1860719

### CODE REVIEW

File: build/tools/Automation/Automation.csproj [hunk 1]
```diff
@@ -22,4 +22,11 @@
     <Folder Include="Commons\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="SmartAnalyzers.CSharpExtensions.Annotations" Version="4.2.8">
```
Comment (daniv-msft) [inline build/tools/Automation/Automation.csproj:26]: It looks like we can just update the `Directory.Build.props` file that contains a reference to this package and will automatically be imported in all projects.

Comment (waliMSFT) [reply build/tools/Automation/Automation.csproj:26]: you are right. thanks for the suggestion

**Review by cormacpayne — DISMISSED** [review]
:shipit:

File: src/BuildScriptGenerator/BuildScriptGenerator.csproj [hunk 2]
```diff
@@ -72,17 +72,17 @@
...
+    <EmbeddedResource Include="Dockerfile.oryx.tpl" WithCulture="false" />
+    <EmbeddedResource Include="DotNetCore\DotNetCoreBashBuildSnippet.sh.tpl" WithCulture="false" />
+    <EmbeddedResource Include="Java\JavaBashBuildSnippet.sh.tpl" WithCulture="false" />
+    <EmbeddedResource Include="Node\NodeBashBuildSnippet.sh.tpl" WithCulture="false" />
+    <EmbeddedResource Include="Hugo\HugoBashBuildSnippet.sh.tpl" WithCulture="false" />
+    <EmbeddedResource Include="Php\PhpBashBuildSnippet.sh.tpl" WithCulture="false" />
+    <EmbeddedResource Include="Golang\GolangBashBuildSnippet.sh.tpl" WithCulture="false" />
+    <EmbeddedResource Include="Python\JupyterNotebookBashBuildSnippet.sh.tpl" WithCulture="false" />
+    <EmbeddedResource Include="Python\PythonBashBuildSnippet.sh.tpl" WithCulture="false" />
+    <EmbeddedResource Include="Ruby\RubyBashBuildSnippet.sh.tpl" WithCulture="false" />
```
Comment (william-msft) [inline src/BuildScriptGenerator/BuildScriptGenerator.csproj:85]: nit: please remove extra space if not required.

File: tests/Oryx.RuntimeImage.Tests/Oryx.RuntimeImage.Tests.csproj [hunk 3]
```diff
@@ -18,7 +18,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
     <PackageReference Include="xunit" Version="2.4.2" />
     <PackageReference Include="Xunit.SkippableFact" Version="1.4.13" />
-    <PackageReference Include="XunitXml.TestLogger" Version="3.0.70" />
+    <PackageReference Include="XunitXml.TestLogger" Version="3.1.11" />
```
Comment (william-msft) [inline tests/Oryx.RuntimeImage.Tests/Oryx.RuntimeImage.Tests.csproj:21]: Should we also update `XunitXml.TestLogger` in the other places too?
image

Comment (william-msft) [reply tests/Oryx.RuntimeImage.Tests/Oryx.RuntimeImage.Tests.csproj:21]: Hi @waliMSFT I noticed you updated another the version for another `XunitXml.TestLogger` in your [recent commit](https://github.com/microsoft/Oryx/pull/2088/commits/692ccbdff41cf95d6603d7a98b0de0c2dbdb005a). But I see total of 5 files being changed. The screenshot above shows there are 7. Do we not need to update the other 2?

Comment (waliMSFT) [reply tests/Oryx.RuntimeImage.Tests/Oryx.RuntimeImage.Tests.csproj:21]: I forgot to reply on your comment.
I probably missed the other files. I have updated the package on all the files now.

**Review by daniv-msft — APPROVED** [review]
Looks good to me! Thanks

**Pull request merged**